cerberus-auth-connector-okta/src/test/java/com/nike/cerberus/auth/connector/okta/OktaApiClientHelperTest.java

```diff
@@ -54,6 +55,17 @@ public void setup() {
   /////////////////////////
   // Test Methods
   /////////////////////////
+  @Test
+  public void oktaApiClientHelper() {
+    OktaConfigurationProperties oktaConfigurationProperties =
+        mock(OktaConfigurationProperties.class);
+    when(oktaConfigurationProperties.getBaseUrl()).thenReturn("url");
+    when(oktaConfigurationProperties.getApiKey()).thenReturn("api_key");
+    this.oktaApiClientHelper = new OktaApiClientHelper(oktaConfigurationProperties);
+    assertNotNull(oktaApiClientHelper);
```

Comment:
This looks like it only tests the mocks in this method and the constructor for OktaApiClientHelper.

Reply:
removed this test